doc/examples/registration/plot_register_affine.py

```diff
@@ -0,0 +1,187 @@
+"""
+=================================================
+Estimate affine transformation to register images
+=================================================
+
+In this example, we use image registration to find an affine transformation
+which can be used to align our target and reference images.
+
+The :func:`skimage.measure.register_affine` function uses a Gaussian pyramid,
+a cost function, and an optimization function to find the affine transformation
+that best aligns a reference image to a target image. This transformation
+(which is expressed as an (ndim+1, ndim+1) matrix) can be used by
+:func:`scipy.ndimage.affine_transform` to convert the target image to the
+reference space. This approach is explained in detail in Chapter 7 of Elegant
+SciPy [1]_.
+
+.. [1] Juan Nunez-Iglesias, Stefan van der Walt, and Harriet Dashnow. Elegant
+       SciPy: The Art of Scientific Python. 1st. O'Reilly Media, Inc.,
+       2017. isbn: 1491922877, 9781491922873.
+
+"""
+
+import numpy as np
+from scipy import ndimage as ndi
+from matplotlib import pyplot as plt
+
+from skimage.data import camera
+from skimage import registration
```

Comment:
Please use new import convention `import skimage as ski` and update function calls accordingly.